simulation/gauss-pgun-job.py

```diff
@@ -13,10 +13,38 @@
 local_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
 sys.path.append(local_dir)
 
-from common import set_tags
+#from common import set_tags
 
+from Configurables import LHCbApp, CondDB
 
-def execute(evt_type=13104021, stereo=5):
+def set_tags(stereo=5):
+    LHCbApp().Simulation = True
+    CondDB().Upgrade = True
+    t = {#"DDDB": "dddb-20140606",
```

Comment:
Remove this as well, and the other commented out stuff of the local tag things below